wwwroot/HttpPlatformHandlerStartup.ps1

```diff
@@ -103,14 +123,17 @@ log('Searching for duplicate plugins.')
 $plugins = Get-ChildItem '.\sonarqube-*\extensions\plugins\*' -Filter '*.jar'
 $pluginBaseName = $plugins | ForEach-Object { $_.Name.substring(0, $_.Name.LastIndexOf('-')) }
 $uniquePlugins = $pluginBaseName | Select-Object -Unique
-$duplicates = Compare-Object -ReferenceObject $uniquePlugins -DifferenceObject $pluginBaseName
-if($duplicates)
+if($uniquePlugins)
```

Comment:
I don't understand this change. We will always have `uniquePlugins` unless there are 0 plugins installed.

Reply:
I don't have any plugins installed or until there are any plugins installed, it is throwing an error every time. It's just defending programming.